### PR TITLE
Feature/wikirunnerCreateDevon4jProject

### LINKED ISSUE #170: wiki runner: createDevon4jProject
Implement the command `createDevon4jProject` in the wiki runners.

**Parameters:**
1. Project name: string

**Wiki Console**
Create a new devon4J project with devon IDE, passing the project name.

**Wiki Editor**
Nothing to do

**Wiki VS Code**
Nothing to do

**Wiki Eclipse**
Describe how to create a new application [directly in Eclipse](https://devonfw.com/website/pages/docs/devon4j.asciidoc_tutorials.html#tutorial-newapp.asciidoc_from-eclipse)

### PR DESCRIPTION
**Pull request by MarcelDiessner**
closes #170

### CODE REVIEW

**Review by EduardKrieger — DISMISSED** [review]

File: runners/wikiEclipse/templates/createDevon4jProject.asciidoc [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+In order to create a new application you must use the archetype provided by devon4j which uses the https://maven.apache.org/guides/introduction/introduction-to-archetypes.html[maven archetype functionality].
+
+To create a new application, you should have installed devonfw IDE. Follow the https://devonfw.com/website/pages/docs/devonfw-ide-introduction.asciidoc.html[devonfw-ide] documentation to install the same.
+
+==== Create the Project
+
+Firstly, create a new Maven Project.
+Then, choose the 'devon4j-template-server' archetype.
+Fill the Group Id, Artifact Id, Version and Package for your project. The package name should be <%= name; %>.
```
Comment (EduardKrieger) [inline runners/wikiEclipse/templates/createDevon4jProject.asciidoc:15]: Maybe try to highlight the labels of some text field like  *Group Id* for better visibility where the user has to insert something.

File: runners/wikiConsole/templates/createDevon4jProject.asciidoc [hunk 2]
```diff
@@ -0,0 +1,42 @@
+== Create a new Devon4j Project
+
+=== Create a new Devon4j Project from command prompt
```
Comment (GuentherJulian) [inline runners/wikiConsole/templates/createDevon4jProject.asciidoc:3]: devon4j and project in lower cases. Also in the rest of the template

File: runners/wikiConsole/templates/createDevon4jProject.asciidoc [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+To create a new application, you should have installed devonfw IDE. Follow the https://devonfw.com/website/pages/docs/devonfw-ide-introduction.asciidoc.html[devonfw-ide] documentation to install the same.
+
+==== Create the Project
+
+Now you can use devonfw to setup a java project for you by executing the following devon command in terminal.
+
+`devon java create <%= name; %>`
+
+==== What is generated
```
Comment (GuentherJulian) [inline runners/wikiConsole/templates/createDevon4jProject.asciidoc:17]: ==== What is generated?

File: runners/wikiConsole/templates/createDevon4jProject.asciidoc [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+The application template (archetype) generates a Maven multi-module project. It has the following modules:
+
+- *api*: module with the API (REST service interfaces, transferobjects, datatypes, etc.) to be imported by other apps as a maven dependency in order to invoke and consume the offered (micro)services.
```
Comment (GuentherJulian) [inline runners/wikiConsole/templates/createDevon4jProject.asciidoc:21]: transfer-objects

File: runners/wikiConsole/templates/createDevon4jProject.asciidoc [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+- *core*: maven module containing the core of the application.
+
+- *batch*: optional module for batch(es)
+
+- *server*: module that bundles the entire app (core with optional batch) as a WAR file.
+
+- *ear*: optional maven module is responsible to packaging the application as a EAR file.
```
Comment (GuentherJulian) [inline runners/wikiConsole/templates/createDevon4jProject.asciidoc:29]: Is an ear module created by default?

File: runners/wikiConsole/templates/createDevon4jProject.asciidoc [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+==== Create the Project
+
+Now you can use devonfw to setup a java project for you by executing the following devon command in terminal.
+
+`devon java create <%= name; %>`
+
+==== What is generated
+
+The application template (archetype) generates a Maven multi-module project. It has the following modules:
```
Comment (GuentherJulian) [inline runners/wikiConsole/templates/createDevon4jProject.asciidoc:19]: Sometimes you write Maven in lowercase letters and sometimes in uppercase letters. You should use the same writing everywhere. On the devonfw page the lower case version is used

**Review by GuentherJulian — CHANGES_REQUESTED** [review]

**Review by GuentherJulian — APPROVED** [review]

**Pull request merged**